index.html

```diff
@@ -8011,7 +8019,7 @@ <h2>Conformance of PNG
 
 <!-- <ol start="1"> --><ol>
 <li>It is able to read any PNG datastream that conforms to this
-International Standard, including both public and private chunks
+specification, including both public and private chunks
```

Comment:
@svgeesus This and other changes were not reverted?

Reply:
Thanks, please check again now